views/index.ejs

```diff
@@ -26,9 +26,12 @@
 	</div>	
 	<div class="modules">
 		<h3>Available modules</h3>
-		<ul id="modulesList">
+		<ul>
 			<li v-for="item in turfModules">
-				<label><input type="checkbox" name="{{item}}" value="{{item}}" v-model="selectedModules" />{{ item }}</label>
+				<div>
+					<label><input type="checkbox" name="{{item}}" value="{{item}}" v-model="selectedModules" />{{item}}</label>
+					<a href="/doc/{{item}}" onclick="event.preventDefault();var popup = window.open('/doc/{{item}}', 'turf-builder', 'width=600,height=400');popup.focus();">(i)</a>
```

Comment:
Instead of sending a request to the backend and opening a popup window with the content. 
What i would recommend is to do the following:
```
<a href="https://www.npmjs.com/package/@turf/{{item}}" target="_blank">(i)</a>
```
This will open the link of the npm documentation for that specific turf package.

Another solution is to open the github specfic turf package page.
E.g. 
```html
<a href="https://github.com/Turfjs/turf/tree/master/packages/turf-{{item}}" target="_blank">(i)</a>
```

This way we avoid another endpoint and the user can quickly see the package documentation